cmd2/cmd2.py

```diff
@@ -835,23 +841,8 @@ def tokens_for_completion(self, line: str, begidx: int, endidx: int) -> Tuple[Li
 
         return tokens, raw_tokens
 
-    # noinspection PyUnusedLocal
-    @staticmethod
-    def basic_complete(text: str, line: str, begidx: int, endidx: int, match_against: Iterable) -> List[str]:
```

Comment:
Should we add more to the CHANGELOG in regards to some of the more minor breaking changes such as moving `basic_complete` from a method in `cmd.Cmd` to a function in `utils`?

Same goes for making all arguments passed the standard four to completer functions keyword only?